### DISCUSSION TITLE
Allowed Amount File and adjustment charges

### DISCUSSION THREAD

**Discussion by jltang-kp** [CMSgov/price-transparency-guide#448] [Q&A]
I was not able to find information on this but if we have an adjusted allowed amount charge would we still publish the original charge amount or just show the adjusted charge?

**Comment by shaselton-usds** [comment]
@jltang-kp Can you speak more to the adjusted charge? Thanks.

**Comment by jltang-kp** [comment]
@shaselton-usds Sure, a scenario that could trigger an adjustment is if a member has a secondary coverage. We may only have one coverage on file when initially processing a claim and get corrected claim info later on for an indemnity coverage. Depending on how the claim was originally paid with we may upwards or downwards adjust the allowed amounts with the secondary coverage info.

Another scenario that could cause adjusted claims is overlapping coverages. Hope this helps.

**Reply by shaselton-usds** [reply]
Sorry for the delayed response. The Allowed Amounts file is meant to reflect the accurate allowed amount charged/reimbursed, so if that is so through the result of adjustments, then those charges should be disclosed in the file. It is understood that adjustments could take place outside of the reporting period and those claims wouldn't need to be reported due to falling outside of the reporting time period requirements.
